packages/cache/src/cache.ts

```diff
@@ -86,23 +86,24 @@ export async function restoreCache(
   }
 
   const compressionMethod = await utils.getCompressionMethod()
+  let archivePath = ''
+  try {
+    // path are needed to compute version
+    const cacheEntry = await cacheHttpClient.getCacheEntry(keys, paths, {
```

Comment:
can the try block be just around the http client to handle errors from the server?

Reply:
I guess the idea here is to ensure that _any_ failure is suppressed so that user does not get blocked due to cache action failures.
A bit scary thought but I guess it would be good for users' workflows to continue executing as in reality there is no blocker due to cache action failure. This may mean that users may not even notice things like archive/unarchive failing, and that's why I feel we should have [monitors](https://github.com/actions/toolkit/pull/1122#discussion_r901311622) to ability to detect any anomaly in such failures.